doc/response-tagging.rst

```diff
@@ -75,21 +74,21 @@ Only ``/one`` will stay in the cache.
 
 .. note::
 
-    Don't forget to call :ref:`flush <flush>` on the ``CacheInvalidator`` to commit the invalidations
-    to the caching proxy.
-
-.. _custom_tags_header:
```

Comment:
removing this leads to "/home/travis/build/FriendsOfSymfony/FOSHttpCache/doc/varnish-configuration.rst:180: WARNING: undefined label: custom_tags_header (if the link has no caption the label must precede a section header)"

please re-add it right before the "Custom Tags Header" subtitle.